packages/client/src/components/InstanceChat/index.tsx

```diff
@@ -151,6 +154,21 @@ const InstanceChat = (props: Props): any => {
     }
   }, [isMultiline])
 
+  useEffect(() => {
+    window.addEventListener('resize', handleWindowResize)
+
+    return () => {
+      window.removeEventListener('resize', handleWindowResize)
+    }
+  }, [])
+
+  const handleWindowResize = () => {
+    setDimensions({
+      height: window.innerHeight,
+      width: window.innerWidth
+    })
+  }
+
```

Comment:
Can we centralize this logic into a feature detection store, rather than repeating it multiple times? Look for where I am detecting whether or not touch events are available.